### ISSUE TITLE
Proof of Concept Requirements Discussion

### ISSUE THREAD

**Issue by randman22222** [randman22222/Pok-mon-Sepia#5]
So my first thoughts are we should have the starting zone with custom sprites, music, text, NPCs, etc. successfully implemented, just to showcase that we are competent. I think we should also all collaborate on the worldbuilding and story writing on the side, so that we have some general direction on where to go after the PoC is assembled.

**Comment by randman22222** [comment]
Anything else you guys think is important for the Proof of Concept?

**Comment by decoyjayman** [comment]
i was reading these out of order so i thought that you were talking about People of Color

**Comment by decoyjayman** [comment]
but yeah, that would be a good thing to work on. maybe we should start a skype group for discussions then post summaries somewhere in here, or make a separate section for discussion summaries on here? however this works

**Comment by randman22222** [comment]
Yeah this isn't a good medium for discussions. I'll have to figure out what medium we're all on...

**Comment by randman22222** [comment]
Okay I think we'll just put a .md file in the master branch for proof of concept requirements. When we make edits and push to commit, we'll have discussions there. It's a slightly better place to discuss stuff, I guess?
